docs/schemas/Image-Based-Spatial-Transcriptomics-Metadata/NIMP Donor-Specimen Metadata.csv

```diff
@@ -0,0 +1,13 @@
+Proposed BICAN Field Name,Aliases,LinkML Class,Definition,Nullable,Permissible Values,Data Type,Data Examples,Min Value,Max Value,Unit,Statistical Variable Type,Subsets,Notes
+tissue sample name,tissue_sample_local_name,Tissue Sample,Block of tissue that was used for further dissection and/or sectioning,FALSE,,text,,,,,categorical,Tracking/Identification,
+brain subdivision,brain_slab/brain_subdivision,Tissue Sample,The subdivision of the brain the tissue sample belongs to,TRUE,[brain_subdivision],ValueSet,cerebrum,,,,categorical,,

```

Comment:
for consistency it should be probably "nimp_brain_subdivision", since that's the name of the file.
the same for the other elements in this file